.semaphore/semaphore.yml

```diff
@@ -8,47 +8,7 @@ agent:
     os_image: "ubuntu2004"
 
 blocks:
-  - name: "Run checks"
```

Comment:
are we just getting rid of running tests in CI?

Reply:
No, they're still run. From the OP:

> - I haven't fully modernized the Seamaphore build, but I did remove the explicit stack step since it's redundant with the `nix-build` and that invokes Cabal anyways.